shub/image/list.py

```diff
@@ -119,6 +118,8 @@ def _run_cmd_in_docker_container(image_name, command, environment):
         client.start(container)
     except docker.errors.APIError as e:
         explanation = e.explanation or ''
+        if isinstance(explanation, binary_type):
+            explanation = explanation.decode('utf-8')
```

Comment:
Since we're using it twice, maybe we can outsource this into a utility function?

```
def ensure_unicode(s, encoding='utf-8'):
    return s.decode('utf-8') if isinstance(s, binary_type) else s
```